docker-compose.yaml

```diff
@@ -16,21 +14,35 @@ services:
       - ALL
     cap_add:
       - NET_BIND_SERVICE
+    security_opt:
+      - no-new-privileges:true
+    logging:
+      driver: json-file
+      options:
+        max-size: "1m"
+        max-file: "1"
 
   redis:
     container_name: redis
-    image: docker.io/library/redis:alpine
-    command: redis-server --save 30 1 --loglevel warning
+    image: cgr.dev/chainguard/valkey:latest
+    command: --save 60 1
```

Comment:
Could you keep "loglevel" to warning? Also please keep "--save" to 60

Reply:
When I tested it at the time it didn't seem to work, so I removed it. I reverted the change.